Animal Competition/Java/PerryThePlatypus.java

```diff
@@ -0,0 +1,42 @@
+public class PerryThePlaypus extends CoolAnimals implements ITraits{
+    
+  public int numberOfLegs() {
+    return 4;
+  }
+  
+  public string species() {
+    return "Platypus";
+  }
+
+  public int numberOfEyes() {
+    return 2;
+  }
+
+  public boolean hasClaws() {
+    return false;
+  }
+
+  public boolean hasTail() {
+    return true;
+  }
+  
+  public boolean fightsCrime(){
+    return true;
+  }
+
+  public boolean canClimb(){
+    return true;
+  }
+
+  public boolean canFly(){
```

Comment:
Similiar to the `black-and-white` comment, can you imagine consideration beyond whether or not a creature can fly?  Like how about it's flying prowest: a number from, say, 0-100 where 0 is can't fly at all and 100 is the best flyer imaginable.